openfecli/tests/commands/test_plan_rbfe_network.py

```diff
@@ -210,9 +210,9 @@ def test_plan_rbfe_network_charge_changes(cdk8_files):
 def custom_yaml_settings():
     return """\
 network:
-  method: generate_minimal_redundant_network
+  method: generate_lomap_network
```

Comment:
happy to add this as a separate test, but if we were only using `generate_minimal_redundant_network` here to make sure the custom yaml was working, we might as well use the lomap network generator here.

Reply:
I used a debugger to make sure lomap was actually being called in this test, but is stored anywhere that we can check in the test? Or is this motivation for better logging?

Reply:
Better logging might work, alternatively you could just could the number of edges being generated - you should end up with more than the N_ligands-1 edges you would normally get with an MST.